app/src/test/scala/org/alephium/explorer/persistence/queries/TransactionQueriesSpec.scala

```diff
@@ -104,7 +104,7 @@ class TransactionQueriesSpec
     val (total, _)           = run(queries.getBalanceAction(address)).futureValue
     val (totalDEPRECATED, _) = run(queries.getBalanceActionDEPRECATED(address)).futureValue
 
-    total is ALPH.alph(3) // ARG!
```

Comment:
What is `ARG` here 😅

Reply:
simulating a complaining sound :sweat_smile: